examples/basics/emag_basics.py

```diff
@@ -3,220 +3,249 @@
 
 Motor-CAD E-magnetic example script
 ===================================
-This example provides a Motor-CAD E-magnetic script. This script
-creates a partial custom winding pattern to change
-parameter values, run the analysis, and plot results. To create
-a full winding pattern, parameters must be specified
-for all coils.
+This example provides a Motor-CAD E-magnetic script to run calculations and plot results.
 """
-
 # %%
+# This script will change model parameter values (geometry, winding pattern and materials),
+# run magnetic calculations, extract and plot results.
+#
+# .. note::
+#    This script creates a partial custom winding pattern. To create a full winding pattern,
+#    parameters must be specified for all coils.
+#
 # Set up example
 # --------------
-# Setting up this example consists of performing imports, specifying the
-# working directory, launching Motor-CAD, and disabling all popup
-# messages from Motor-CAD.
+# To set up the example, perform imports, specify the working directory, launch Motor-CAD, and
+# disable all popup messages from Motor-CAD.
 #
 # Perform required imports
 # ~~~~~~~~~~~~~~~~~~~~~~~~
+# Import ``pymotorcad`` to access Motor-CAD. Import ``pyplot`` from ``matplotlib`` to display
```

Comment:
```suggestion
# Import ``pymotorcad`` to access Motor-CAD. Import ``pyplot`` from Matplotlib to display
```
Use the product name for Matplotlib to avoid a code entity of a code entity?